src/main/java/duke/commands/FindCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke.commands;
+
+import java.util.List;
+
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+
+/**
+ * FindCommand represents a command to find a list of tasks based
+ * on similarity of a given string.
+ */
+public class FindCommand extends Command {
+
+    private Ui ui;
+    private TaskList taskList;
+    private String string;
+
+    /**
+     * Creates a FindCommand to find all tasks on the Task list
+     * based on similarity of a given string.
+     * @param ui The ui used.
```

Comment:
Same suggestion as previous comments, there are other places with the same behaviour too.